api/metadata_endpoints.py

```diff
@@ -30,11 +37,40 @@ def get_landing_page(request):
     )
 
 
-def get_collection_metadata(request) -> Collection:
+async def get_collection_metadata(request, is_self) -> Collection:
+    # TODO: Try to remove/lower duplication with /locations endpoint
+    ts_request = dstore.GetObsRequest(temporal_mode="latest")
+    ts_response = await get_obs_request(ts_request)
+
+    # Sadly, this is a different parameter as in the /locations endpoint, due to an error in the EDR spec
+    # See: https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/issues/427
+    all_parameters: Dict[str, Parameter] = {}
+
+    for obs in ts_response.observations:
+        parameter = Parameter(
+            description=obs.ts_mdata.title,
```

Comment:
Sorry I did not catch this earlies. But I do not think `ts_mdata.title` can be used here, as it will most likely be different for every time series. What we really need is the standard_name description from nerc vocab.

Reply:
We also run in to the same issue in `/locations`

Reply:
So you are hitting the "mutiple definitions" 500 error?

Shall we just use `standard_name` for description for now?
I am guessing that `obs.ts_mdata.instrument` for label (3 lines down) is also not correct.

Reply:
To get back to this, I don't think the description from the nerc vocab would do, as that is a description of the standard_name, not of this specific parameter.

All the different air_temperatures now have the same description, which doesn't make to much sense. Some measure it at grass height, some at 2.0m. And there can have different average times. The would all be described as "Air temperature is the bulk temperature of the air, not the surface (skin) temperature.".

What we need in the metadata is a field that describes the parameter as measured, independent of the station.

Reply:
BTW, why are all the `ts_mdata.title` different in your case. Do you include the station name in them?